### PR TITLE
Centos project does not support custom kernels

### PR DESCRIPTION
**Pull request by SvenDowideit**
- see #9696

Signed-off-by: Sven Dowideit SvenDowideit@docker.com

Docker-DCO-1.1-Signed-off-by: Sven Dowideit SvenDowideit@docker.com (github: SvenDowideit)

@jperrin - are you happy for me to say it this way?

@vbatts should I add something similar for RHEL?

@unclejack @fredlf @jamtur01

### CODE REVIEW

**Comment by phemmer** [comment]
Probably should specify which CentOS version, as CentOS 7 doesn't use 2.6.32

**Comment by SvenDowideit** [comment]
@phemmer you're right - It would be good to know - will RHEL/Centos be stuck on 3.10 forever, or is that shell game going to change?

File: docs/sources/installation/centos.md [hunk 1]
```diff
@@ -27,6 +27,13 @@ simply run the following command.
 
     $ sudo yum install docker
 
+## Kernel support
+
+Currently the CentOS project will only support Docker via EPEL package when running
```
Comment (jamtur01) [inline docs/sources/installation/centos.md]: via the

**Comment by jamtur01** [comment]
LGTM broadly.

**Comment by kdomanski** [comment]
LGTM

**Comment by unclejack** [comment]
@SvenDowideit This looks good to me and I think we need to say something similar about RHEL.

I think we also need to say something specific about RHEL 6 and CentOS 6 because that's where people usually want a newer kernel. Should they really want a newer kernel, they should upgrade to RHEL 7 or CentOS 7, depending on what they're using and what they want to use.

**Comment by kdomanski** [comment]
@SvenDowideit Judging by history, it will be stuck on 3.10 with possible minor backports.

**Comment by vbatts** [comment]
LGTM. And yes, RHEL and most linux distros will not support issues when using a custom kernel ...

**Comment by fredlf** [comment]
LGTM

**Pull request merged**